modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -550,6 +554,99 @@ export default class I3SConverter {
     }
   }
 
+  /**
+   * Find property table in tile
+   * For example it can be batchTable for b3dm files or property table in gLTF extension.
+   * @param sourceTile
+   */
+  private getPropertyTable(sourceTile: TileHeader): FeatureTableJson | null {
```

Comment:
Move all the propertyTable logic on geometry-converter level because it is part of the content

Reply:
Done